composer.json

```diff
@@ -86,7 +86,10 @@
     "fix-cs": [
       "php ./vendor/bin/phpcbf"
     ],
-    "phpstan": ["phpstan analyze --ansi --memory-limit=1G"]
+    "phpstan": ["phpstan analyze --ansi --memory-limit=1G"],
+    "post-install-cmd": [
```

Comment:
This runs `npm ci && npm run build` after composer deps are installed, so that the GraphiQL App is built and ready to use.